### ISSUE TITLE
Accessibility: Low color contrast on blue text elements

### ISSUE THREAD

**Issue by vvvt** [corona-warn-app/cwa-website#93]
in addition to  #16

## Where to find the issue
Blue text elements throughout the page

`<h5>` and `<span class="enumeration">` on
https://www.coronawarn.app/

`<a>` on
https://www.coronawarn.app/de/
https://www.coronawarn.app/en/
https://www.coronawarn.app/de/#privacy
https://www.coronawarn.app/en/#privacy
https://www.coronawarn.app/de/community
https://www.coronawarn.app/en/community
https://www.coronawarn.app/de/imprint/

## Describe the issue
The given elements have the text color `#1294d4`, which has a contrast ratio of only **3.38:1** to the `#ffffff` background. Since this falls under the category _small text_ it should have at least **4.5:1** to meet the WCAG AA Recommendations

Additionally, in the section `<section id="privacy">` the link color `#1294d4` is paired with the  background color `#edf2f7`, resulting in an even lower contrast ratio of **3:1**. Again, since this is _small text_, a ratio of at least **4.5:1** is desirable to meet WCAG AA.

## Suggested change
Adapt the color scheme to meet WCAG AA contrast ratios or - if possible - WCAG AAA contrast ratios. The WCAG specifications can be read [here](https://www.w3.org/TR/WCAG21/#contrast-minimum).
